task/06-conditions-n-loops-tasks.js

```diff
@@ -373,10 +418,20 @@ function toNaryString(num, n) {
  *   ['/web/favicon.ico', '/web-scripts/dump', '/webalizer/logs'] => '/'
  */
 function getCommonDirectoryPath(pathes) {
-    throw new Error('Not implemented');
+  let splitPatth = pathes.map((item) => item.split("/"));
```

Comment:
Pay attention to param naming (I think you mean "splitPath" here :) ) Thank you!